rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -575,6 +575,12 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 	u32 errDialog, PSetList setList, PSetBuf setBuf, PFuncList funcList, PFuncFixed funcFixed, PFuncStat funcStat,
 	PFuncFile funcFile, u32 container, u32 unk_op_flags /*TODO*/, vm::ptr<void> userdata, u32 userId, PFuncDone funcDone)
 {
+	if (CellSaveDataSetList list{}; setList.try_read(list))
```

Comment:
The second overload try_read(void) seems best suited for this because you do not have to spell its type.

Reply:
done